### PR TITLE
Fix issue502

### PR DESCRIPTION
**Pull request by orrmany**
Proposing single character bugfix in cores/nRF5/avr/pgmspace.h:105 for issue #502

### CODE REVIEW

**Comment by zjwhitehead** [comment]
Thats it! 
Was having this exact issue and just tested it again. Can confirm this fixes it! 👍

File: cores/nRF5/avr/pgmspace.h [hunk 1]
```diff
@@ -102,7 +102,7 @@ typedef const void* uint_farptr_t;
 #define pgm_read_word(addr) (*(const unsigned short *)(addr))
 #define pgm_read_dword(addr) (*(const unsigned long *)(addr))
 #define pgm_read_float(addr) (*(const float *)(addr))
-#define pgm_read_ptr(addr) (*(const void *)(addr))
+#define pgm_read_ptr(addr) (*(const void* *)(addr))
```
Comment (hathach) [inline cores/nRF5/avr/pgmspace.h:105]: Yeah, it is spot on with samdcore https://github.com/arduino/ArduinoCore-samd/blob/master/cores/arduino/avr/pgmspace.h#L106

You should also file an issue on this original repo as well to help them out https://github.com/sandeepmistry/arduino-nRF5/blob/master/cores/nRF5/avr/pgmspace.h#L105

**Comment by zjwhitehead** [comment]
Would be great to get this as a part of a new release soon!
Lots of other big things that would make it worth doing https://github.com/adafruit/Adafruit_nRF52_Bootloader/compare/0.3.2...master

File: cores/nRF5/avr/pgmspace.h (hunk 1 above)
Comment (orrmany) [reply cores/nRF5/avr/pgmspace.h:105]: Agree, I've made https://github.com/sandeepmistry/arduino-nRF5/pull/418

**Pull request merged**